ext/node/02_require.js

```diff
@@ -915,6 +916,9 @@
   window.__bootstrap.internals = {
     ...window.__bootstrap.internals ?? {},
     require: {
+      setNodeModulesDir() {
```

Comment:
Nitpick: This function name sounds like someone needs to provide a path. Maybe `markUsesNodeModulesDir` or something along those lines? I think everything else is fine in the PR.